src/test/java/org/openmrs/contrib/qaframework/page/AllergyPage.java

```diff
@@ -0,0 +1,27 @@
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+
+public class AllergyPage extends Page {
```

Comment:
once you have sorted out my previous feedback, please add all possible functionality onto the respective page. thanks